ivy/array/__init__.py

```diff
@@ -275,11 +275,7 @@ def __lt__(self, other):
 
     @_native_wrapper
     def __le__(self, other):
-        other = to_native(other)
-        res = ivy.builtin_le(self._data, other)
-        if res is NotImplemented:
-            return res
-        return to_ivy(res)
+        return ivy.less_equal(self._data, other)
```

Comment:
I couldn't use `ivy.builtin_le` because that fails one of the tensorflow Array API tests